gcloud/bigquery/job.py

```diff
@@ -869,6 +869,7 @@ class _AsyncQueryConfiguration(object):
     _flatten_results = None
     _priority = None
     _use_query_cache = None
+    _user_defined_function_resources = None
```

Comment:
https://travis-ci.org/GoogleCloudPlatform/gcloud-python/builds/145240341#L464

Per the pylint defaults (which we don't override), this class attribute name is too long:  it wants a maximum of 30 characters.  We could spell it `_udf_resources` (and change the property accordingly.